apps/tup-cms/src/taccsite_cms/templates/snippets/js-new-tab-new-window-accessibility.html

```diff
@@ -0,0 +1,23 @@
+
+<script id="new-tab-new-window-accessibility" type="text/javascript">
+
+    // NOTE: Some links have `target="_blank"` even though they should not
+    const pathsOfManualNewTabLinks = document.querySelectorAll(
+        '[target="_blank"]:not([aria-describedby])'
+    );
+
+    // Adds accessibility attribute to links that open in new tab
+    function supportA11y( link ) {
+        const hasA11yNote = link.querySelector('span.sr-only');
+        const hasA11yDesc = link.hasAttribute('aria-describedby');
```

Comment:
**Possibly Redundant Code.** Links that already have `aria-describedby` seem to be excluded twice. (Likely a mistake in the source code you were given to reference). If it is indeed redundant, then consider improving upon this code by choosing which means of exclusion to rely on:

- via CSS selector i.e. `:not([aria-describedby])`
- or via JavaScript conditional `hasA11yDesc `

<details><summary>Direction</summary>

- Which is easier for others to understand?
- Is either drastically more efficient than the other?

</details>